drivers/SmartThings/zwave-switch/profiles/inovelli-dimmer-power-energy.yml

```diff
@@ -17,6 +16,14 @@ components:
     version: 1
   categories:
   - name: Switch
+- id: LEDColorConfiguration
+  capabilities:
+  - id: colorControl
```

Comment:
don't you need to remove this capability from `main` as well, like you did below?

Reply:
Yes! Thanks for catching it.
Fixed.